Maestro/subscriptions.json

```
@@ -1025,13 +1023,12 @@
         }
       }
     },
-    // Automate opening PRs to merge aspnet release/3.1 branches back to master and main.
+    // Automate opening PRs to merge aspnet release branches back to master and main
+    // Aspnetcore-Tooling is explicitly only release/3.1 to avoid the various release/vs* branches
     {
       "triggerPaths": [
-        "https://github.com/dotnet/aspnetcore/blob/release/3.1//**/*",
-        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1//**/*",
-        "https://github.com/dotnet/blazor/blob/release/3.1//**/*",
-        "https://github.com/dotnet/ef6/blob/release/6.4//**/*"
+        "https://github.com/dotnet/aspnetcore/blob/release//**/*",
```

We've had cases in the past where `release/*` branches were created accidentally. To avoid useless PRs in that case and because I am not sure `**` works for the second part of the branch name, suggest
```suggestion
        "https://github.com/dotnet/aspnetcore/blob/release/2.1//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/3.1//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/5.0//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/6.0*//**/*",
```